### ISSUE TITLE
MB_PASSWORD_COMPLEXITY/MB_PASSWORD_LENGTH not being respected

### ISSUE THREAD

**Issue by blakeblackshear** [metabase/metabase#3283]
I am unable to get metabase to recognize _some_ environment variable settings. Specifically, `MB_COLORIZE_LOGS` and `MB_PASSWORD_COMPLEXITY` seem to be ignored.

I have tried the following with no luck:
```
MB_COLORIZE_LOGS=false java -jar ~/Downloads/metabase.jar
java -Dmb.colorize.logs=false -jar ~/Downloads/metabase.jar
```

I have tried the following versions `0.19.3`, `0.17.0`, and `0.15.0` without any luck.

**Comment by camsaul** [comment]
Hi @blakeblackshear I can't reproduce the problem on my local computer. Are you using the standard OS X `Terminal.app` with `bash`?


### Running Normally

![screen shot 2016-08-30 at 2 55 10 pm](https://cloud.githubusercontent.com/assets/1455846/18108606/d33029d8-6ec1-11e6-942d-5faff7e160bc.png)


### Running w/ `MB_COLORIZE_LOGS`
 
![screen shot 2016-08-30 at 2 55 24 pm](https://cloud.githubusercontent.com/assets/1455846/18108605/d32fc4e8-6ec1-11e6-8a0a-81a2140cc425.png)

**Comment by blakeblackshear** [comment]
It is removing those colors, but not the white highlighting. This causes it to show up in syslog like so:
```
 08-30 21:19:57 #033[1mINFO db.migrations#033[0m :: Running all necessary data migrations, this may take a minute.
```

I also had my fingers crossed that it would remove the emojis as well.

I guess this works as designed, and I am really submitting a feature request. 😄 

However, I have not been able to get the `MB_PASSWORD_COMPLEXITY`  or `MB_PASSWORD_LENGTH` setting to work. Do these perhaps not apply to administrators?

**Comment by camsaul** [comment]
The `#033[1m` and `#033[0m` you're seeing are actually added by log4j based on our logging configuration. Our log4j properties file is [here](https://github.com/metabase/metabase/blob/master/resources/log4j.properties); if you want to tweak it, you can use that as a starting point and then tell log4j to use the custom one instead by setting the Java prop`log4j.configuration`, e.g.

```bash
java -Dlog4j.configuration=file:/path/to/custom/log4j.properties -jar metabase.jar
```

What values are you using for `MB_PASSWORD_COMPLEXITY` / `MB_PASSWORD_LENGTH`?

**Comment by blakeblackshear** [comment]
`strong` and `10`

**Comment by camsaul** [comment]
Ok, I'll take a look and see if I can reproduce it.

**Comment by NeckBeardPrince** [comment]
Any update to this?

**Comment by camsaul** [comment]
Fixed by #3593
